### PR TITLE
Use Specs to use Terra.describeViewports

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
Updates to use Terra.describesViewports().

- Uncovered a few previously invalid screenshots 
- Skipped terra-popup. That will need its own overhaul / cleanup

### CODE REVIEW

File: packages/terra-application-layout/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +16,7 @@ Unreleased
 
 ### Changed
 * Update tests for terra-toolkit v5 and terra-dev-site v5 changes
+* Update wdio specs to use Terra.describeViewports
```
Comment (StephenEsser) [inline packages/terra-application-layout/CHANGELOG.md]: Double entry?

File: packages/terra-date-picker/CHANGELOG.md [hunk 2]
```diff
@@ -11,7 +11,11 @@ Unreleased
 
 ### Changed
 * Update tests for terra-toolkit v5 and terra-dev-site v5 changes
+<<<<<<< HEAD
```
Comment (yuderekyu) [inline packages/terra-date-picker/CHANGELOG.md]: oops

File: packages/terra-notification-dialog/CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ Unreleased
 
 ### Changed
 * Update tests for terra-toolkit v5 and terra-dev-site v5 changes
+<<<<<<< HEAD
```
Comment (yuderekyu) [inline packages/terra-notification-dialog/CHANGELOG.md]: oops

File: packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js [hunk 4]
```diff
@@ -1,13 +1,10 @@
-const viewports = Terra.viewports('medium');
-
 // Color contrast will be resolved in https://github.com/cerner/terra-core/issues/1670
 const ignoredA11y = {
```
Comment (yuderekyu) [inline packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js]: not in scope of this pr, but is this disabled rule still necessary?

File: packages/terra-date-picker/CHANGELOG.md (hunk 2 above)
Comment (mjhenkes) [inline packages/terra-date-picker/CHANGELOG.md]: remove merge conflict markers

File: packages/terra-notification-dialog/CHANGELOG.md (hunk 3 above)
Comment (mjhenkes) [inline packages/terra-notification-dialog/CHANGELOG.md]: Remove merge conflict markers

File: packages/terra-tabs/CHANGELOG.md [hunk 5]
```diff
@@ -4,18 +4,22 @@ ChangeLog
...
 * Update tests for terra-toolkit v5 and terra-dev-site v5 changes
 
+6.2.0 - (June 12, 2019)
+------------------
 ### Added
 * box-shadow theme variable for in-active tab: --terra-tabs-structural-box-shadow
 * z-index theme variable for active tab: --terra-tabs-structural-active-z-index
 
+### Changed
+* Update tests for terra-toolkit v5 and terra-dev-site v5 changes
```
Comment (mjhenkes) [inline packages/terra-tabs/CHANGELOG.md]: is there a mis-merge here?

File: packages/terra-notification-dialog/CHANGELOG.md (hunk 3 above)
Comment (emilyrohrbough) [reply packages/terra-notification-dialog/CHANGELOG.md]: noooo. I tried to merge online and then it just wasn't an issue. I guess it must have just marked all as "good". Will work on fixing these. (how embarrassing 😅)

File: packages/terra-date-picker/CHANGELOG.md (hunk 2 above)
Comment (emilyrohrbough) [reply packages/terra-date-picker/CHANGELOG.md]: sure you don't want these in here?? 😄

File: packages/terra-application-layout/CHANGELOG.md (hunk 1 above)
Comment (emilyrohrbough) [reply packages/terra-application-layout/CHANGELOG.md]: fixed: https://github.com/cerner/terra-framework/pull/729/commits/e93d1ec71368775f94249cc8536355234696b2a6

File: packages/terra-notification-dialog/CHANGELOG.md (hunk 3 above)
Comment (emilyrohrbough) [reply packages/terra-notification-dialog/CHANGELOG.md]: fixed: https://github.com/cerner/terra-framework/pull/729/commits/0b7264816a20a234bae654707f72b1c8f1f16230

File: packages/terra-date-picker/CHANGELOG.md (hunk 2 above)
Comment (emilyrohrbough) [reply packages/terra-date-picker/CHANGELOG.md]: fixed: https://github.com/cerner/terra-framework/pull/729/commits/0b7264816a20a234bae654707f72b1c8f1f16230

File: packages/terra-notification-dialog/CHANGELOG.md (hunk 3 above)
Comment (emilyrohrbough) [reply packages/terra-notification-dialog/CHANGELOG.md]: fixed: https://github.com/cerner/terra-framework/pull/729/commits/0b7264816a20a234bae654707f72b1c8f1f16230

File: packages/terra-tabs/CHANGELOG.md (hunk 5 above)
Comment (emilyrohrbough) [reply packages/terra-tabs/CHANGELOG.md]: fixed: https://github.com/cerner/terra-framework/pull/729/commits/eb13cb5cbec7a5c6633d9bcbb60a5407dac2e60c

File: packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js (hunk 4 above)
Comment (emilyrohrbough) [reply packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js]: I just removed it and tried. Receiving the following error: 
```
running chrome
AssertionError: expected no accessibility violations but got:
	{
  "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
  "help": "Elements must have sufficient color contrast",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=axeAPI",
  "id": "color-contrast",
  "impact": "serious",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-hour-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Hours</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeEmpty > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(2) > label[for=\"terra-time-hour-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-minute-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Minutes</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeEmpty > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(4) > label[for=\"terra-time-minute-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-hour-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Hours</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(2) > label[for=\"terra-time-hour-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-minute-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Minutes</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#datetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(4) > label[for=\"terra-time-minute-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-hour-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Hours</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#disabledDatetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(2) > label[for=\"terra-time-hour-input\"]"
      ]
    },
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#ffffff",
            "contrastRatio": 3.97,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#808080",
            "fontSize": "9.0pt",
            "fontWeight": "normal"
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<html lang=\"en\" dir=\"ltr\">",
              "target": [
                "html"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<label for=\"terra-time-minute-input\" class=\"TimeInput-module__mobile-input-label___2W6me\">Minutes</label>",
      "impact": "serious",
      "none": [],
      "target": [
        "#disabledDatetimeValueProvided > .DateTimePicker-module__time-facade___RbC0N > .TimeInput-module__mobile-time-picker___3RdnY > .TimeInput-module__time-input-group___9V3oL:nth-child(4) > label[for=\"terra-time-minute-input\"]"
      ]
    }
  ],
  "tags": [
    "cat.color",
    "wcag2aa",
    "wcag143"
  ]
}
    at Object.runAccessibilityTest (/Users/er047227/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/accessibility.js:23:35)
    at Context.<anonymous> (/Users/er047227/terra-framework/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/validate-element.js:77:28)
    at Promise (<anonymous>)
    at F (/Users/er047227/terra-framework/node_modules/babel-runtime/node_modules/core-js/library/modules/_export.js:36:28)
```

Comment (emilyrohrbough) [reply packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js]: Logged this issue: https://github.com/cerner/terra-framework/issues/744

**Pull request merged**